src/midgard/logging.cc

```diff
@@ -136,6 +136,7 @@ class StdOutLogger : public Logger {
   }
   virtual void Log(const std::string& message, const std::string& custom_directive = " [TRACE] ") {
 #ifdef __ANDROID__
+    std::string tmp = custom_directive; // to prevent -Wunused-parameter
     __android_log_print(ANDROID_LOG_INFO, "valhalla", "%s", message.c_str());
```

Comment:
i was going to suggest maybe actually using the parameter, seems kind of lame to add a no-op to the code just to shut up -Werror. we could add the custom directive to the error message in the android version:

```suggestion
    __android_log_print(ANDROID_LOG_INFO, "valhalla", "%s %s", custome_directive.c_str(), message.c_str());
```

Reply:
Actually, I wanted to do that too.
But in my opinion, it's not a familiar log format for Android (because there is a TAG method).

therefore,
I wanted to respect the intent of the original author of the Android log output part.

Even since I am new to the valhalla community, I wanted to respect the intentions of others.

If a lot of people agree, I agree with the amendment.

Reply:
I dont feel strongly about it. We can merge what you have

Reply:
I just confirmed that you are the original author, and I'm confused. haha.

I'm glad that someone else doesn't have the same troubles as me on Android builds.